src/website/views/templates/fieldset.pug

```diff
@@ -0,0 +1,218 @@
+
+p.-text Basic example of several types of form elements in one fieldset.
+
+.example.-mb--3
+  .chi-app.-py--3.-bg--grey-20(style="border-radius:.15em .15em 0 0;")
+    .a-grid__container
+      .a-grid.-noGutter.-b--1.-p--3.-bg--white
+        div.a-col.-w--12
+          h5.-mb--2 Business Information
+          .a-inputWrapper.-floatingLabel.-mb--1#bname
+            input(type='text', class='a-input -xl', id='name')
+            label(for='name') Business name
+        div.a-col.-w--12
+          .a-inputWrapper.-floatingLabel#bphone
+            input(type='text', class='a-input -xl', id='phone')
+            label(for='phone') Business phone
+        div.a-col.-w--12
+          h6.-mt--2 Business address
+          .a-inputWrapper.-floatingLabel.-mb--1#baddress1
+            input(type='text', class='a-input -xl', id='address1')
+            label(for='address1') Address line 1
+          .a-inputWrapper.-floatingLabel.-mb--1#baddress2
+            input(type='text', class='a-input -xl', id='address2')
+            label(for='address2') Address line 2
+        div.a-col.-w--6.-mr--1
+          .a-inputWrapper.-floatingLabel#bcity
+            input(type='text', class='a-input -xl', id='city')
+            label(for='city') City
+        div.a-col.-w--4.-mr--1
+          .a-inputWrapper.-floatingLabel#bstate
+            select#floating-label-select.a-input.-xl
+              option
+              option Option 1
+              option Option 2
+              option Option 3
+            label(for="floating-label-select") State
+        div.a-col
+          .a-inputWrapper.-floatingLabel#bzip
+            input(type='text', class='a-input -xl', id='zip')
+            label(for='zip') Zip
+        div.a-col.-w--12
+          .m-formGroup.-my--2
+            input(type="checkbox", class="a-input", id="checkbox1" checked)
+            label(for="checkbox1")
+            label.-text(for="checkbox1") This has been my business's address for more than one year
+        div.a-col.-w--12
+          h6 Business contact
+        div.a-col.-mr--1
+          .a-inputWrapper.-floatingLabel#bcname
+            input(type='text', class='a-input -xl', id='contact_name')
+            label(for='contact_name') Name
+        div.a-col
+          .a-inputWrapper.-floatingLabel#bctitle
+            input(type='text', class='a-input -xl', id='contact_title')
+            label(for='contact_title') Title
+        div.a-col.-w--12.-mt--1
+          .a-inputWrapper.-floatingLabel#bcemail
+            input(type='text', class='a-input -xl', id='email')
+            label(for='email') Email
+        div.a-col.-w--12
+          h6.-mt--2 Executive contact
+        div.a-col.-w--12
+          .m-formGroup.-mb--2
+            input(type="checkbox", class="a-input", id="checkbox2" checked)
+            label(for="checkbox2")
+            label.-text(for="checkbox2") Same as business contact
+        div.a-col.-w--12
+          h6.-mt--2 Additional business information
```

Comment:
Remove `-mt--2` - it's setting a margin that's inconsistent with the space between the checkbox and h6 above it.

Reply:
done